src/sbt-test/universal/absolute-path/build.sbt

```diff
@@ -0,0 +1,21 @@
+enablePlugins(JavaAppPackaging)
```

Comment:
Test looks good. However can you split this into two tests?

The bash one goes into `sbt-test/bash` and ( I know this is confusing :( ) into `sbt-test/windows`